packages/components/nodes/memory/ZepMemory/ZepMemory.ts

```diff
@@ -0,0 +1,130 @@
+import { SystemChatMessage } from 'langchain/schema'
+import { INode, INodeData, INodeParams } from '../../../src/Interface'
+import { getBaseClasses } from '../../../src/utils'
+import { ZepMemory, ZepMemoryInput } from 'langchain/memory/zep'
+import { ICommonObject } from '../../../src'
+
+class ZepMemory_Memory implements INode {
+    label: string
+    name: string
+    description: string
+    type: string
+    icon: string
+    category: string
+    baseClasses: string[]
+    inputs: INodeParams[]
+
+    constructor() {
+        this.label = 'Zep Memory'
+        this.name = 'ZepMemory'
+        this.type = 'ZepMemory'
+        this.icon = 'memory.svg'
+        this.category = 'Memory'
+        this.description = 'Summarizes the conversation and stores the memory in zep server'
+        this.baseClasses = [this.type, ...getBaseClasses(ZepMemory)]
+        this.inputs = [
+            {
+                label: 'Base URL',
+                name: 'baseURL',
+                type: 'string',
+                default: 'http://127.0.0.1:8000'
+            },
+            {
+                label: 'Auto Summary',
+                name: 'autoSummary',
+                type: 'boolean',
+                default: true
+            },
+            {
```

Comment:
I think we can also still have the session id in case when user want to resume the previous conversation. if the session id field is not filled, then we can use the first `chatid`:

```
{
      label: 'Session Id',
      name: 'sessionId',
      type: 'string',
      optional: true,
      additionalParams: true
},
```

Reply:
This is better than the current, I'll implement it later! 😀